src/questions/checkingQuestions.ts

```diff
@@ -11,6 +11,11 @@ export const checkingProof1: Question[] = [
     prompt: "Explain your reasoning in 1 sentence.",
     type: QuestionType.Text,
   },
+  {
```

Comment:
define once, use everywhere :) this prompt is used in 3 places in this file alone, and multiple times in completeQuesiton.tsx as well. make it a variable and copy-paste that instead